test/organization/is_worker.js

```diff
@@ -57,6 +57,7 @@ contract('Organization.isWorker()', async (accounts) => {
 
     // Dummy Transaction to increase block number.
     for (let i = 0; i < deltaExpirationHeight; i += 1) {
+      // eslint-disable-next-line no-await-in-loop
```

Comment:
To fix **no-await-in-loop** linting error,add `"no-await-in-loop": "off"` under rules in `.eslintrc.json`.

Reply:
We should not disable rules in .eslintrc file. I've only disabled `no-await-in-loop` in places that we broke the rule. However, we should not add new breaking cases and fix all `no-await-in-loop` errors properly (in some cleanup round) for all repos.